### DISCUSSION TITLE
Question about the imputation returns negative values.

### DISCUSSION THREAD

**Discussion by sunxi822** [amices/mice#414] [Getting started] [answered]
Hello everyone,

I am trying to impute missing data by using `mice( )`.
It works well, but it returns negative values for a variable that should be positive.
I don't know how to solve this problem.

Hope for anyone's reply.
Many thanks!

**Comment by gerkovink** [comment, marked as answer]
Imputation method `mice.impute.pmm()` will never return values that are not observed, so using that method should solve your problem.
